cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -22,7 +24,7 @@ spec:
     spec:
       serviceAccountName: elasticsearch-logging
       containers:
-      - image: gcr.io/google_containers/elasticsearch:v2.4.1-2
+      - image: gcr.io/google_containers/elasticsearch-k8s-s3:5.5.0-11
```

Comment:
having s3 included in the default is a bit tough as mentioned above.  Its not a useful plugin for GKE clusters.